apiExamples/roundedRightAlign.html

```diff
@@ -0,0 +1,4 @@
+<auro-button aria-label="in-flight" rounded iconOnly id="rightAlignElem" style="float: right;">
+  Back to Top
```

Comment:
Use the same text for the button as the right align style.

Reply:
Changes were made under this commit: [docs(rounded): add examples for rounded button](https://github.com/AlaskaAirlines/auro-button/pull/214/commits/3ba91aa69416d83385db790b3cde6264023b3f42) https://github.com/AlaskaAirlines/auro-button/issues/212